packages/api/tests/endpoints/workflows.js

```diff
@@ -73,3 +107,70 @@ test('CUMULUS-912 GET with pathParameters and with an unauthorized user returns
     assertions.isUnauthorizedUserResponse(t, response);
   });
 });
+
+test.serial('GET with no path parameters and an authorized user returns a list of workflows', async (t) => {
+  const request = {
+    httpMethod: 'GET',
+    headers: authHeaders
+  };
+
+  return testEndpoint(workflowsEndpoint, request, (response) => {
+    t.is(response.statusCode, 200);
+    const results = JSON.parse(response.body);
+
+    t.deepEqual(results, workflowList);
+  });
+});
+
+test.serial('GET an existing workflow with an authorized user returns a specific workflow', async (t) => {
+  const request = {
+    httpMethod: 'GET',
+    pathParameters: {
+      name: 'HelloWorldWorkflow'
+    },
+    headers: authHeaders
+  };
+
+  return testEndpoint(workflowsEndpoint, request, (response) => {
+    t.is(response.statusCode, 200);
+    const result = JSON.parse(response.body);
+
+    t.deepEqual(result, workflowList[0]);
+    t.is(result.template, 's3://bucket/cumulus/workflows/HelloWorldWorkflow.json');
+  });
+});
+
+test.serial('GET with path parameters returns a 404 for a nonexistent workflow', async (t) => {
+  const request = {
+    httpMethod: 'GET',
+    pathParameters: {
+      name: 'NonexistentWorkflow'
+    },
+    headers: authHeaders
+  };
+
+  return testEndpoint(workflowsEndpoint, request, (response) => {
+    t.is(response.statusCode, 404);
+
+    const result = JSON.parse(response.body);
+    t.is(result.message, 'The specified key does not exist.');
```

Comment:
I wonder if it would be worth it to throw a different error type above and check against the error type instead of the error message. Not really necessary though